projects/core/src/lib/talks/talks-page/talks-page.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="talks">
```

Comment:
Please, use the same name as component name, it should be `jsmu-talks-page`, then you can create elements from it like `jsmu-talks-page__talk`